### PR TITLE
@bentley/imodeljs-native 2.13.0

### PR DESCRIPTION
**Pull request by pmconne**

### CODE REVIEW

**Review by grigasp — APPROVED** [review]

**Review by kabentley — APPROVED** [review]

**Review by bsteinbk — APPROVED** [review]

**Review by mgooding — APPROVED** [review]

File: docs/changehistory/NextVersion.md [hunk 1]
```diff
@@ -35,6 +35,50 @@ To adjust the limit after startup, assign to [TileAdmin.gpuMemoryLimit]($fronten
 
 This feature replaces the `@alpha` `TileAdmin.Props.mobileExpirationMemoryThreshold` option.
 
+## IModelHost and IModelApp Initialization Changes
+
+Initialization processing of iModel.js applications, and in particular the order of individual steps for frontend and backend.js classes has been complicated and vague, involving several steps that vary depending on application type and platform. This release attempts to clarify and simplify that process, while maintaining backwards compatibility. In general, if your code uses [IModelHost.startup]($backend)` and [IModelApp.startup]($frontend) for web visualization, it will continue to work without changes. However, for native (desktop and mobile) apps, some refactoring may be necessary. See [IModelHost documentation]($docs/learning/backend/IModelHost.md) for appropriate backend initialization, and [IModelApp documentation]($docs/learning/frontend/IModelApp.md) for frontend initialization.
```
Comment (calebmshafer) [inline docs/changehistory/NextVersion.md:40]: Did you mean to reference `backend.js` in the first sentence?

Comment (calebmshafer) [reply docs/changehistory/NextVersion.md:40]: Also a floating back tick after `[IModelHost.startup]($backend)`

**Review by calebmshafer — COMMENTED** [review]
Can you remove the `core/mobile-manager/CHANGELOG.*` files?  They'll break rush change during a publish.

File: core/backend/src/CodeSpecs.ts [hunk 2]
```diff
@@ -6,8 +6,8 @@
  * @module Codes
  */
 
-import { DbResult, Id64, Id64String, Logger } from "@bentley/bentleyjs-core";
-import { CodeScopeSpec, CodeSpec, IModelError, IModelStatus } from "@bentley/imodeljs-common";
+import { DbResult, Id64, Id64String, IModelStatus, Logger } from "@bentley/bentleyjs-core";
```
[the remark this answers is not in the document]
Comment (kabentley) [reply core/backend/src/CodeSpecs.ts:9]: i didn't change anything. We always exported `IModelStatus` from two places. I just made it consistent in `backend` that we always import from `bentleyjs-core`

**Comment by kabentley** [comment]
> Can you remove the `core/mobile-manager/CHANGELOG.*` files? They'll break rush change during a publish.

done

File: core/backend/src/IModelHost.ts [hunk 3]
```diff
@@ -197,17 +190,18 @@ export class IModelHostConfiguration {
   };
 
   /**
-   * Application (host) type
-   * @alpha
+   * Application (host) type for native logging
+   * @internal
    */
-  public applicationType?: ApplicationType;
+  public applicationType?: IModelJsNative.ApplicationType;
```
[the remark this answers is not in the document]
Comment (kabentley) [reply core/backend/src/IModelHost.ts:196]: done

File: docs/changehistory/NextVersion.md (hunk 1 above)
Comment (kabentley) [reply docs/changehistory/NextVersion.md]: both fixed now

File: core/frontend/src/IModelApp.ts [hunk 4]
```diff
@@ -134,6 +133,7 @@ export interface IModelAppOptions {
    * @internal
    */
   featureToggles?: FeatureToggleClient;
+  rpcInterfaces?: RpcInterfaceDefinition[];
```
[the remark this answers is not in the document]
Comment (kabentley) [reply core/frontend/src/IModelApp.ts:136]: yes, you can now pass your rpcInterfaces to the `xxxApp.startup` method (see [here](https://github.com/imodeljs/imodeljs/blob/08d39487c0e9e647de6c227dfb8fd21c7c79d3ca/test-apps/display-test-app/src/frontend/App.ts#L143) for example).

it was never clear before what order you had to register rpc vs startup.

[the remark this answers is not in the document]
Comment (kabentley) [reply core/frontend/src/IModelApp.ts:136]: the old way still works, btw

File: core/backend/src/CodeSpecs.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (calebmshafer) [reply core/backend/src/CodeSpecs.ts:9]: Ah okay, didn't realize we exported it from two separate packages.

File: core/frontend/src/IModelApp.ts (hunk 4 above)
[the remark this answers is not in the document]
Comment (calebmshafer) [reply core/frontend/src/IModelApp.ts:136]: Sounds good, this is similar to the way the iTwin Viewer started exposing it as well.  The initial instantiation of the viewer would take in the list of RpcProps and RpcInterfaces as part of the configuration.

cc: @kckst8 fyi of this incoming change.

**Pull request merged**